### PR TITLE
Reply comments

### PR DESCRIPTION
**Pull request by andreldsa**
Comment is a new directive, to simplify the view handle, once comment has a list of replies and actions specific to it's object.

It was added a button to cleanup all notifications, as well as the removal of the Facebook right panel to optimize the system performance (the panel was executing lots of requests per second to sync with Facebook database).

### CODE REVIEW

File: admin.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 import urllib
 import hashlib
 
+from utils import Utils
```
Comment (mayzabeel) [inline admin.py]: testes

File: admin.py [hunk 2]
```diff
@@ -104,10 +113,10 @@ def delete_all_in_index(index):
 class BaseHandler(webapp2.RequestHandler):
     """Base Handler."""
 
-    def handle_exception(self, exception, debug):
-        """Exception."""
-        logging.error(str(exception))
-        self.response.write("oops! %s\n" % str(exception))
+    # def handle_exception(self, exception, debug):
```
[the remark this answers is not in the document]
Comment (andreldsa) [reply admin.py]: done

File: app/home/home.html [hunk 3]
```diff
@@ -46,15 +46,25 @@
...
+            <div class="md-toolbar-tools" flex layout="row" layout-align="start center">
+              Eventos
+            </div>
+          </md-toolbar>
+          <md-card-content>
+            <md-grid-list
+              md-cols="5" md-cols-gt-md="5"
+              md-row-height="1:1"
+              md-gutter="4px" md-gutter-gt-sm="4px">
+              <md-grid-tile ng-repeat="institution in homeCtrl.followingInstitutions" ng-click="homeCtrl.goToInstitution(institution.key)"
```
Comment (mayzabeel) [inline app/home/home.html]: apaga essa lista

File: app/post/postDetailsDirective.js [hunk 4]
```diff
@@ -379,6 +348,113 @@
...
+                function success() {
+                    delete commentCtrl.comment.replies[reply.id];
+                    MessageService.showToast('Comentário excluído com sucesso');
+                }, function error(response) {
+                    MessageService.showToast(response.data.msg);
+                }
+            );
+        };
+
+        commentCtrl.deleteComment = function deleteComment() {
```
Comment (mayzabeel) [inline app/post/postDetailsDirective.js]: Colocar TODO de verificação, se comentário tem replies não pode ser removido

Comment (andreldsa) [reply app/post/postDetailsDirective.js]: o html só mostra o botão de remover se não tiver replies, e o backend tem verificação para não deixar remover nesse caso.

**Pull request merged**